inst/components/sidebar.scss

```diff
@@ -1,8 +1,11 @@
 $bslib-sidebar-padding: $spacer * 1.5 !default;
 $bslib-sidebar-icon-size: $spacer !default;
-
+$bslib-sidebar-bg: var(--bslib-sidebar-bg, #{$gray-200}) !default;
```

Comment:
```suggestion
$bslib-sidebar-bg: $gray-200 !default;
$bslib-sidebar-fg: color-contrast($bslib-sidebar-bg) !default;
```

Reply:
I'm going to have to come back to this next week. If we set the background color of the sidebar, then we need to set the parent layout container colors to get the right animation. And then we need to set the main content area colors by default too. In which case, our problem would then shift to the R side where we'd need a way to unset everything such that the whole sidebar layout could inherit from its parent.

In other words, I need to think about this some more.

Reply:
I applied the above change, which now works with the updated sidebar markup